src/screens/ContactScreen/elements/Search.js

```diff
@@ -27,15 +28,16 @@ const Search = ({onPress, animatedValue, onChangeText, text, onClearText, isLoad
         onSubmitEditing={onPress}
         textAlignVertical="center"
         blurOnSubmit={true}
+        keyboardAppearance="dark"
       />
       <View style={styles.wrapperIcon}>
-        <MemoIc_search width={20} height={20} />
+        <MemoIc_search width={20} height={20} fill={COLORS.gray300} />
       </View>
       <View style={styles.wrapperSecondaryIcon}>
         {isLoading && <ActivityIndicator style={styles.loader} color={COLORS.blackgrey} />}
         {!isLoading && (
           <Pressable onPress={onClearText}>
-            <IcClearCircle width={20} height={20} />
+            <IconClose width={8} height={8} color={COLORS.gray400} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `keyboardAppearance` prop has been added to the `TextInput` component, which sets the theme of the keyboard. Also, the `MemoIc_search` and `IconClose` components now have additional props such as `width`, `height`, and `fill` or `color`. Make sure these changes are consistent with your design system and the colors are accessible.
```diff
+ keyboardAppearance="dark"
+ <MemoIc_search width={20} height={20} fill={COLORS.gray300} />
+ <IconClose width={8} height={8} color={COLORS.gray400} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->